### PR TITLE
feat: add new global message component on purchase confirmation screen

### PR DESCRIPTION
**Pull request by reidzeibel**
context: https://github.com/AtB-AS/kundevendt/issues/17075#issuecomment-2014785178

Straightforward PR, tried to add rule, but @gorandalum mentioned that it should be a different context. Suggestions to context naming are appreciated. :)

### CODE REVIEW

File: src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx [hunk 1]
```diff
@@ -453,6 +453,14 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             isMarkdown={true}
           />
         )}
+        <GlobalMessage
+          style={styles.purchaseInformation}
+          globalMessageContext={GlobalMessageContextEnum.appPurchaseInformation}
+          textColor="secondary"
```
Comment (gorandalum) [inline src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx:459]: Is `textColor="secondary"` correct? 

From Figma:
Screenshot 2024-04-02 at 11 24 31

File: src/global-messages/types.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export enum GlobalMessageContextEnum {
   appTicketing = 'app-ticketing',
   appPurchaseOverview = 'app-purchase-overview',
   appPurchaseConfirmation = 'app-purchase-confirmation',
+  appPurchaseInformation = 'app-purchase-information',
```
Comment (gorandalum) [inline src/global-messages/types.ts:14]: Hard to intuitively understand where 'app-purchase-information' is located in the app. Maybe something like 'app-purchase-confirmation-bottom' would be more descriptive?

Comment (reidzeibel) [reply src/global-messages/types.ts]: Updated

File: src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx (hunk 1 above)
Comment (reidzeibel) [reply src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx]: Updated, was using non-subtle global message, so the color change wasn't noticeable. Thanks for pointing it out!

**Review by rosvik — APPROVED** [review]

**Review by marius-at-atb — APPROVED** [review]

**Pull request merged**